migrations/V79__Update_Operational_Needs.sql

```diff
@@ -0,0 +1,79 @@
+/* Rename old key name */
+ALTER TYPE OPERATIONAL_NEED_KEY RENAME TO OPERATIONAL_NEED_KEY_OLD;
```

Comment:
Can you remove the old enum at the end of this migration?

Reply:
Thanks for catching this Clay! Yes, I'll drop this and update the keys elsewhere.

Reply:
Updated! I ended up skipping the rename and dropped the type earlier instead.